doc/STYLE_GUIDE.md

```diff
@@ -58,6 +58,10 @@
 * References to constructor functions should use PascalCase
 * References to constructor instances should be camelCased
 * References to methods should be used with parentheses: `socket.end()` instead of `socket.end`
+* To draw special attention to a note, adhere to the following guidelines:
```

Comment:
To be quite honest, I generally prefer the `*Note*:` style and have been incrementally working towards that for quite some time. I guess this is fine tho. Mildly -0